repos/system_upgrade/el8toel9/actors/checkcustomnetworkscripts/libraries/customnetworkscripts.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from leapp import reporting
+
+CUSTOM_NETWORK_SCRIPTS = [
+    "/sbin/ifup-local",
+    "/sbin/ifup-pre-local",
+    "/sbin/ifdown-local",
+    "/sbin/ifdown-pre-local",
+]
+DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
```

Comment:
Sorry for maybe stupid question - this link is broken because there is no such article yet and will be updated once it appears?

Reply:
The link is reserved for publishing the article, and the Docs Team decide to publish it around RHEL-9 GA in the next spring.

Reply:
@fernflower that's expected. the link will be generated just ~days before the release. We can expect this to be common for el8toel9 until RHEL 9 GA.